verification/Spreadsheets/MINTp0_BasicTests.csv

```diff
@@ -1,55 +1,55 @@
-MintController.sol,removeMinter,controllers[msg.sender]==M,bt011,bt011 removeMinter sets controllers[msg.sender] to 0
-MintController.sol,removeMinter,minterManager = 0,bt030,bt030 removeMinter reverts when minterManager is 0
-MintController.sol,removeMinter,minterManager != MinterManagerInterface,bt031,bt031 removeMinter reverts when minterManager is a user account
-MintController.sol,removeMinter,minterManager == OK,bt032,bt032 removeMinter works when minterManager is ok
-MintController.sol,removeMinter,minterManager.isMinter == false,bt041,bt041 removeMinter(M) works when minterManager.isMinter(M)=false
-MintController.sol,removeMinter,minterManager.isMinter == true,bt042,bt042 removeMinter(M) works when minterManager.isMinter(M)=true
-MintController.sol,removeMinter,minterManager.minterAllowance[M] == 0,bt053,bt053 removeMinter works when the minterAllowance is 0
-MintController.sol,removeMinter,minterManager.minterAllowance[M] == X,bt054,bt054 removeMinter works when the minterAllowance is not zero
-MintController.sol,removeMinter,minterManager.minterAllowance[M] == BIG,bt055,bt055 removeMinter works when the minterAllowance is big
-MintController.sol,setMinterManager,msg.sender == owner,bt006,bt006 setMinterManager works when owner is msg.sender
-MintController.sol,setMinterManager,msg.sender != owner,bt007,bt007 setMinterManager reverts when owner is not msg.sender
-MintController.sol,setMinterManager,minterManager == 0,bt025,bt025 setMinterManager(x) works when existing minterManager = 0
-MintController.sol,setMinterManager,minterManager != 0,bt026,bt026 setMinterManager(x) works when existing minterManager != 0
-MintController.sol,setMinterManager,minterManager == msg.sender,bt027,bt027 setMinterManager(x) works when x = msg.sender
-MintController.sol,setMinterManager,minterManager == newMinterManager,bt028,bt028 setMinterManager(x) works when x = minterManager
-Ownable.sol,constructor,msg.sender == owner,bt001,bt001 Constructor - owner is msg.sender
-Ownable.sol,transferOwnership,msg.sender == owner,bt002,bt002 transferOwnership works when owner is msg.sender
-Ownable.sol,transferOwnership,msg.sender != owner,bt003,bt003 transferOwnership reverts when owner is not msg.sender
\ No newline at end of file
+File,Function,Special Condition,Code,Description
+Controller.sol,configureController,msg.sender == owner,bt004,bt004 configureController works when owner is msg.sender
+Controller.sol,configureController,msg.sender != owner,bt005,bt005 configureController reverts when owner is not msg.sender
+Controller.sol,configureController,controllers[C]=0,bt019,bt019 configureController works when controller[C]=0
+Controller.sol,configureController,controllers[C] != 0,bt020,bt020 configureController works when controller[C] != 0
+Controller.sol,configureController,controllers[C]=C,bt021,"bt021 configureController(C,C) works"
```

Comment:
probably want to remove quotes around descriptions in this csv for consistency

Reply:
removed all quotes

Reply:
Returned quotes because they are needed.  If the description has a comma inside it, then the entire column contents needs to be in quotation marks.